utils/demo/helpers/initDemo.js

```diff
@@ -49,15 +49,19 @@ export async function peerImport(moduleId) {
   }
 
   if (moduleId === '@icr/polyseg-wasm') {
-    return import('@icr/polyseg-wasm');
+    return import(
+      /* webpackChunkName: "icr-polyseg-wasm" */ '@icr/polyseg-wasm'
```

Comment:
Can you update with the vite ignore and the separate variable name?

Reply:
These are inside our demo which we don't build and don't ship